Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -0,0 +1,147 @@
+
+    /// Sets "IPHONEOS_DEPLOYMENT_TARGET" to `value`
+    func iPhoneOSDeploymentTarget(_ value: String) -> SettingsDictionary {
+        merging(with: ["IPHONEOS_DEPLOYMENT_TARGET": value.asSettingValue])
+    }
+
+    /// Sets "SDKROOT" to "iphoneos"
+    func iPhoneOSasSDKRoot() -> SettingsDictionary {
+        merging(with: ["SDKROOT": "iphoneos"])
+    }
+
+    /// Sets "SDKROOT" to `value`
+    func sdkRoot(_ value: String) -> SettingsDictionary {
+        merging(with: ["SDKROOT": value.asSettingValue])
+    }
+
+    // MARK: - Code signing
+
+    /// Sets "CODE_SIGN_STYLE" to "Manual"
+    func manualCodeSigning() -> SettingsDictionary {
+        merging(with: ["CODE_SIGN_STYLE": "Manual"])
+    }
+
+    /// Sets "CODE_SIGN_STYLE" to "Automatic" and "DEVELOPMENT_TEAM" to `devTeam`
+    func automaticCodeSigning(devTeam: String) -> SettingsDictionary {
+        merging(with: [
+            "CODE_SIGN_STYLE": "Automatic",
+            "DEVELOPMENT_TEAM": devTeam.asSettingValue
+        ])
+    }
+
+    ///Sets "CODE_SIGN_IDENTITY" to "Apple Development"
+    func codeSignIdentityAppleDevelopment() -> SettingsDictionary {
+        codeSignIdentity("Apple Development")
+    }
+
+    ///Sets "CODE_SIGN_IDENTITY" to `identity`
+    func codeSignIdentity(_ identity: String) -> SettingsDictionary {
+        merging(with: ["CODE_SIGN_IDENTITY": identity.asSettingValue])
+    }
+
+    // MARK: - Versioning and Product Name
+
+    ///Sets "PRODUCT_NAME" to `name`
+    func productName(_ name: String) -> SettingsDictionary {
+        merging(with: ["PRODUCT_NAME": name.asSettingValue])
+    }
+
+    /// Sets "CURRENT_PROJECT_VERSION" to `version`
+    func currentProjectVersion(_ version: String) -> SettingsDictionary {
+        merging(with: ["CURRENT_PROJECT_VERSION": version.asSettingValue])
+    }
+
+    /// Sets "VERSIONING_SYSTEM" to "apple-generic"
+    func appleGenericVersioningSystem() -> SettingsDictionary {
+        merging(with: ["VERSIONING_SYSTEM": "apple-generic"])
+    }
+
+    /// Sets "VERSION_INFO_PREFIX" to `version`. If prefix is not `nil`, it's used as "VERSION_INFO_PREFIX"; and suffix as "VERSION_INFO_SUFFIX"
+    func versionInfo(_ version: String, prefix: String? = nil, suffix: String? = nil) -> SettingsDictionary {
+        var versionSettings: SettingsDictionary = ["VERSION_INFO_STRING": version.asSettingValue]
+        versionSettings["VERSION_INFO_PREFIX"] = prefix?.asSettingValue
+        versionSettings["VERSION_INFO_SUFFIX"] = suffix?.asSettingValue
+
+        return merging(with: versionSettings)
+    }
+
+    // MARK: - Swift Settings
+
+    ///Sets "SWIFT_VERSION" to `version`
+    func swiftVersion(_ version: String) -> SettingsDictionary {
+        merging(with: ["SWIFT_VERSION": version.asSettingValue])
+    }
+
+    ///Sets "OTHER_SWIFT_FLAGS" to `flags`
+    func otherSwiftFlags(_ flags: String) -> SettingsDictionary {
+        merging(with: ["OTHER_SWIFT_FLAGS": flags.asSettingValue])
```

Comment:
Maybe it'd be better to change `String` to `String...`? That way it becomes more flexible. You can then just do `flags.joined()`

Reply:
@fortmarek great idea. I did actually `flags.joined(separator: " ")`